tests/test_06_l2vpn_return_codes.py

```diff
@@ -382,7 +381,7 @@ def test_050_create_l2vpn_with_valid_bw(self):
             ],
             "qos_metrics": {
                 "min_bw": {
-                    "value": 11
+                    "value": 10
```

Comment:
I didnt understand this change, Gretel. Can you please elaborate more on this change?